cppcache/src/DistributedSystemImpl.cpp

```diff
@@ -143,11 +147,24 @@ void DistributedSystemImpl::unregisterCliCallback(int appdomainId) {
   }
 }
 
-void DistributedSystemImpl::setThreadName(const std::string& threadName) {
+std::string DistributedSystemImpl::getThreadName(std::thread::id id) {
+  std::string threadName = m_threadNames[id];
+  if (threadName == "") {
+    std::stringstream ss;
+    ss << id;
+    threadName = ss.str();
+  }
+  return threadName;
+}
+
+void DistributedSystemImpl::setThreadName(const std::string& threadName,
```

Comment:
Same comment regarding thread ID as for getThreadName, IMHO I don't see a reason why you would need to pass a thread ID here

Reply:
Good catch.